### ISSUE TITLE
Multiple DS18B20 Support

### ISSUE THREAD

**Issue by budpi** [mholgatem/ThermOS#7]
Would it be too difficult to add support for multiple DS18B20's?

Forgive me for I haven't thought this through completely, but I would imagine you'd have to print out an average between all sensors hooked up to the Raspberry Pi. Or, you could have the ability to select between the individual sensors to act as the master temperature probe and the automation would act accordingly...

Any thoughts?

Thank you very much for you hard work on this project. I am currently using this as a main driver for my furnace.

**Comment by mholgatem** [comment]
ok, the code has been modified so that it will read all ds18b20's and then average them. You will need to update. In order to update without losing your settings, do the following:

```
cd ~
wget goo.gl/tpkyWg -O git_updater.sh
sudo bash git_updater.sh
```

when prompted, enter: **/home/pi/ThermOS**

finally

`sudo bash ~/ThermOS/update.sh -noupdate`

**Comment by budpi** [comment]
The Update worked great for the multiple DS18b20 temp sensors. I have four of them placed throughout my house in each room. I managed to use unused wires from my telephone wires and then mounted the DS18b20's cleanly within the face plate. After the update however, I seem to have an issue with the Outdoor Weather App, after clicking it I get an Internal Server Error, any thoughts? The only thing I changed; I removed the WiFi dongle and have the pi on Ethernet now. 

Thanks again!
